sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/PerfCounterItemCounts.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Internals
+{
+    /// <summary>
+    /// Thread-safe counters for performance metrics.
+    /// </summary>
+    internal sealed class PerfCounterItemCounts
+    {
+        private long _totalRequestCount = 0;
+        private long _totalExceptionCount = 0;
+
+        /// <summary>
+        /// Atomically increments the total request count.
+        /// </summary>
+        /// <returns>The new value after incrementing.</returns>
+        public long IncrementRequestCount()
+        {
+            return Interlocked.Increment(ref _totalRequestCount);
+        }
+
+        /// <summary>
+        /// Atomically increments the total exception count.
+        /// </summary>
+        /// <returns>The new value after incrementing.</returns>
+        public long IncrementExceptionCount()
+        {
+            return Interlocked.Increment(ref _totalExceptionCount);
```

Comment:
Exception rate should be derived from `process.runtime.dotnet.exceptions.count` metrics from the `System.Runtime` meter by calculating count / 60 to get exceptions per second.